addons/addon-base-raas/packages/base-raas-services/lib/alb/alb-service.js

```diff
@@ -338,6 +337,80 @@ class ALBService extends Service {
     // return auditWriterService.write(requestContext, auditEvent);
     return auditWriterService.writeAndForget(requestContext, auditEvent);
   }
+
+  /**
+   * Method to modify rule. The method modify the rule using the ALB SDK client.
+   * Tags are read form the resolvedVars so the billing will happen properly
+   *
+   * @param requestContext
+   * @param resolvedVars
+   * @returns {Promise<D & {$response: Response<D, E>}>}
+   */
+  async modifyRule(requestContext, resolvedVars) {
+    const subdomain = this.getHostname(resolvedVars.prefix, resolvedVars.envId);
+    try {
+      const cidrLen = resolvedVars.cidr.length;
+      // ModifyRule does not accept the empty value to update
+      // so the system should validate the cidr and if it is empty then
+      // replace the default ip "0.0.0.0/0"
+      if (cidrLen === 0) {
+        resolvedVars.cidr = ['0.0.0.0/0'];
+      }
```

Comment:
Defaulting to `0.0.0.0` would be a security concern. Can we throw an exception instead?

Reply:
Yes. Removed the default ip address 0.0.0.0 and added the appropriate error message in the exception.